metricbeat/metricbeat.yml

```diff
@@ -21,6 +21,25 @@ metricbeat.config.modules:
 
 #==================== Elasticsearch template setting ==========================
 
+# Example config for templates
+# 
+# When `setup.template.templates` is configured, the old configuration is ignored.
+#setup.template.name: "metricbeat"
+#setup.template.pattern: "metricbeat*"
+#setup.template.fields: "${path.config}/fields.yml"
+#setup.template.enabled: false 
+#setup.template.overwrite: true
+setup.template.templates:
+  - name: "metricbeat-mix"
+    pattern: "*"
+    modules: "aerospike,apache,system"
```

Comment:
Could we name this more generic. Perhaps call it `fields`: ["transaction"]` instead. This matches better the APM case I would hope.

Not sure if I would already introduce support for a list. We could start with just supporting one entry and migrating to a list later on if needed should not be a breaking change.